### PR TITLE
Use page objects for user-skills-input-item-test

### PR DESCRIPTION
**Pull request by sarupbanskota**
Progress on: #777 
Also under #697

### CODE REVIEW

**Comment by sarupbanskota** [comment]
I discovered while checking out Travis complaints that there doesn't seem to be a component called `page-object-user-skills-input-items`. Should I remove their test files then?

Ref: https://github.com/code-corps/code-corps-ember/tree/develop/app/components

File: tests/pages/components/user-skills-input-items.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import { hasClass, triggerable } from 'ember-cli-page-object';
-
-export default {
-  scope: '.user-skills-input-items',
-
-  listItemIsSelected : hasClass('selected', 'li'),
-  mouseenter         : triggerable('mouseenter', 'li'),
-  mousedown          : triggerable('mousedown', 'li')
-};
```
Comment (sarupbanskota) [inline tests/pages/components/user-skills-input-items.js]: yep, gotcha

File: tests/pages/components/user-skills-input-item.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { collection, hasClass, triggerable } from 'ember-cli-page-object';
+
+export default {
+  scope: '.skill-dropdown-item',
+
+  listItemIsSelected: hasClass('selected'),
+  mouseenter: triggerable('mouseenter'),
+  mousedown: triggerable('mousedown'),
```
Comment (sarupbanskota) [inline tests/pages/components/user-skills-input-item.js]: ahh, that explains why several of my PR tests failed. thanks!

File: tests/integration/components/user-skills-input-item-test.js [hunk 3]
```diff
@@ -25,50 +25,74 @@ moduleForComponent('user-skills-input-item', 'Integration | Component | user ski
...
+    }}
+  `);
 
   assert.equal(page.listItemIsSelected, true);
 
-  let html = '<strong>Ru</strong>by <strong>on</strong> <strong>R</strong>ails';
-  assert.equal(this.$('a').html().trim(), html);
+  assert.equal(page.highlightedStrings(0).text, 'Ru');
+  assert.equal(page.highlightedStrings(1).text, 'on');
+  assert.equal(page.highlightedStrings(2).text, 'R');
```
Comment (sarupbanskota) [inline tests/integration/components/user-skills-input-item-test.js]: agreed, i'll extend this elsewhere too

File: tests/integration/components/user-skills-input-item-test.js [hunk 4]
```diff
@@ -25,50 +25,74 @@ moduleForComponent('user-skills-input-item', 'Integration | Component | user ski
       }
     });
     page.setContext(this);
+  },
+  afterEach() {
+    page.removeContext();
```
Comment (sarupbanskota) [inline tests/integration/components/user-skills-input-item-test.js]: yes sir!

File: tests/integration/components/user-skills-input-item-test.js (hunk 3 above)
Comment (sarupbanskota) [inline tests/integration/components/user-skills-input-item-test.js]: totally. gotcha

**Comment by sarupbanskota** [comment]
@JoshSmith I've understood now. For this PR, can I go ahead and squash the commits on this branch?

**Comment by sarupbanskota** [comment]
actually I accidentally did a rebase and force pushed here 😞 

I'll squash it now, the connection between the commits and review is lost aw

**Comment by JoshSmith** [comment]
@sarupbanskota rebase and squash when you can and merge at will! 👍

**Comment by JoshSmith** [comment]
🙌

**Pull request merged**